oceannavigator/frontend/src/components/DatasetSelector.jsx

```diff
@@ -382,12 +382,17 @@ class DatasetSelector extends React.Component {
         options = this.state.datasetVariables;
       }
 
+      // Work-around for when someone selected a plot that requires
+      // 3D variables, but the selected dataset doesn't have any LOL.
+      // This check prevents a white-screen crash.
+      const stillHasVariablesToShow = options.length > 0;
```

Comment:
Instead of a white-screen crash, we'll force a server-side error so we get a more contextually useful message like so 
![image](https://user-images.githubusercontent.com/5572045/143960145-02d7d8b7-cd09-4745-87e7-d317c03e0821.png)